test/integration/batchSpec.ts

```diff
@@ -40,112 +44,117 @@ describe("Batch/Payment Integration", () => {
   }
 
   it("basic create", async () => {
+    const { nockDone } = await nockBack('create.json')
```

Comment:
I would probably create some sort of wrapper/helper that can record any API call automatically without the need to explicitly declare this constant

I would do this so that the rest of tests can reuse the functionality 🥳